cfgov/unprocessed/css/molecules/global-eyebrow.less

```diff
@@ -109,10 +117,16 @@
             padding-bottom: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
         }
 
-        .m-global-eyebrow_languages {
-            padding-left: 37px; //22px flag + 15px spacing
+        .m-global-eyebrow_right {
```

Comment:
Sure, I was stuck on a good name. Would `body` work even if the tagline isn't in there?